ExampleApp iOS/flowforms/Screens/Form/FormManager.swift

```diff
@@ -0,0 +1,253 @@
+      return nil
+    }
+  }
+
+  var confirmedPasswordErrorMessage: String? {
+    switch confirmPasswordStatus {
+    case StatusCodes.shared.MATCH_UNSATISFIED:
+      return LocalizedString.FormManager.passwordsDontMatchError
+    case StatusCodes.shared.REQUIRED_UNSATISFIED:
+      return LocalizedString.FormManager.passwordConfirmationRequiredError
+    case StatusCodes.shared.MIN_LENGTH_UNSATISFIED:
+      return LocalizedString.FormManager.passwordMinLengthError
+    default:
+      return nil
+    }
+  }
+  
+  var emailVerificationInProgress: Bool {
+    return emailStatus == StatusCodes.shared.IN_PROGRESS
+  }
+  
+  var formValid: Bool {
+    return formStatus == StatusCodes.shared.CORRECT
+  }
+}
+
+// MARK: Bindings
+extension FormManager {
+  
+  var termsAccepted: Binding<Bool> {
+    formModel.form.bindSwitch(
+      field: formModel.termsAccepted,
+      id: FormModel.companion.TERMS_ACCEPTED
+    ) {
+      self.formModel.termsAccepted = $0
+      self.objectWillChange.send()
+    }
+  }
+  
+  var name: Binding<String> {
+    formModel.form.bindField(
+      formModel.name,
+      id: FormModel.companion.NAME
+    ) {
+      self.formModel.name = $0
+      self.objectWillChange.send()
+    }
+  }
+  
+  var email: Binding<String> {
+    formModel.form.bindField(
+      formModel.email,
+      id: FormModel.companion.EMAIL
+    ) {
+      self.formModel.email = $0
+      self.objectWillChange.send()
+    }
+  }
+  
+  var password: Binding<String> {
+    formModel.form.bindField(
+      formModel.password,
+      id: FormModel.companion.PASSWORD
+    ) {
+      self.formModel.password = $0
+      self.objectWillChange.send()
+    }
+  }
+  
+  var confirmPassword: Binding<String> {
+    formModel.form.bindField(
+      formModel.confirmPassword,
+      id: FormModel.companion.CONFIRM_PASSWORD
+    ) {
+      self.formModel.confirmPassword = $0
+      self.objectWillChange.send()
+    }
+  }
+  
+  func configureBindings() {
+    formModel.form.bindStatus(withPublisher: &$formStatus)
+    
+    formModel.form
+      .fieldFor(id: FormModel.companion.NAME)
+      .bindStatus(withPublisher: &$nameStatus)
+    formModel.form
+      .fieldFor(id: FormModel.companion.EMAIL)
+      .bindStatus(withPublisher: &$emailStatus)
+    formModel.form
+      .fieldFor(id: FormModel.companion.PASSWORD)
+      .bindStatus(withPublisher: &$passwordStatus)
+    formModel.form
+      .fieldFor(id: FormModel.companion.CONFIRM_PASSWORD)
+      .bindStatus(withPublisher: &$confirmPasswordStatus)
+  }
+}
+
+// MARK: FFCFlowForm
+extension FlowForm {
+  
+  func bindStatus(withPublisher publisher: inout Published<String>.Publisher) {
+    let formPublisher = createPublisher(for: status)
+    
+    formPublisher
+      .receive(on: DispatchQueue.main)
+      .map({ status in
+        return status.code
+      })
+      .replaceError(with: StatusCodes.shared.INCORRECT)
+      .assign(to: &publisher)
+  }
+  
+  func bindField(
+    _ field: String,
```

Comment:
I would use fieldNamed just to be more clear that we are going to find a field using his name.